debugger/ptraceLib/inferior_status.h

```diff
@@ -4,8 +4,8 @@
 #ifndef _OE_INFERIOR_STATUS_H_
 #define _OE_INFERIOR_STATUS_H_
 
-#include <openenclave/bits/types.h>
 #include <pthread.h>
+#include <stdint.h>
 
 typedef enum _oe_inferior_flags
```

Comment:
Can we remove "_oe" from this type now, for upstream contribution?

Reply:
Done.